lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -197,7 +197,7 @@ def install_puppet(opts = {})
 
             # Certain install paths may not create the config dirs/files needed
             on host, "mkdir -p #{host['puppetpath']}" unless host[:type] =~ /aio/
-            on host, "echo '' >> #{host.puppet['hiera_config']}"
+            #on host, "echo '' >> #{host.puppet['hiera_config']}"
```

Comment:
This should probably be changed to a `printf` or something that works on both rather than commented out.